nd4j/nd4j-backends/nd4j-backend-impls/nd4j-native/src/main/java/org/nd4j/nativeblas/Nd4jCpu.java

```diff
@@ -1058,7 +1099,7 @@ private native void allocate(@Const Pointer hostBuffer,
         public native int _blasMinorVersion(); public native Environment _blasMinorVersion(int setter);
         public native int _blasPatchVersion(); public native Environment _blasPatchVersion(int setter);
 
-        public static native @ByRef Environment getInstance();
+        public native @ByRef Environment getInstance();
```

Comment:
@saudet  that objectify  is corrupting others

Reply:
That just looks like someone tried to modify a generated file. You can ignore that.

Reply:
well, it causes mvn compilation failure. I'll check it again

Reply:
I will recheck again maybe it's related to Vs Code remote
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.7.0:compile (default-compile) on project nd4j-native: Compilation failure: Compilation failure:
[ERROR] /home/rgurbanov/dj/deeplearning4j/nd4j/nd4j-backends/nd4j-backend-impls/nd4j-native/src/main/java/org/nd4j/linalg/cpu/nativecpu/ops/NativeOpExecutioner.java:[127,36] non-static method getInstance() cannot be referenced from a static context
[ERROR] /home/rgurbanov/dj/deeplearning4j/nd4j/nd4j-backends/nd4j-backend-impls/nd4j-native/src/main/java/org/nd4j/linalg/cpu/nativecpu/ops/NativeOpExecutioner.java:[1911,36] non-static method getInstance() cannot be referenced from a static context
[ERROR] /home/rgurbanov/dj/deeplearning4j/nd4j/nd4j-backends/nd4j-backend-impls/nd4j-native/src/main/java/org/nd4j/linalg/cpu/nativecpu/ops/NativeOpExecutioner.java:[1916,40] non-static method getInstance() cannot be referenced from a static context
[ERROR] /home/rgurbanov/dj/deeplearning4j/nd4j/nd4j-backends/nd4j-backend-impls/nd4j-native/src/main/java/org/nd4j/linalg/cpu/nativecpu/ops/NativeOpExecutioner.java:[1992,36] non-static method getInstance() cannot be referenced from a static context
[ERROR] /home/rgurbanov/dj/deeplearning4j/nd4j/nd4j-backends/nd4j-backend-impls/nd4j-native/src/main/java/org/nd4j/linalg/cpu/nativecpu/CpuEnvironment.java:[29,90] non-static method getInstance() cannot be referenced from a static context

```

Reply:
alright, I try to manually objectify(false) others, for now

Reply:
Ok, this could be caused by your macros that have different definitions. Don't try to redefine macros that are already defined. That's never a good idea.

Reply:
sure let me check it first then

Reply:
strangely, it did not work either. I also thought maybe macros definitions in the header causing it. (that header is not include anywhere , so I was ok with using the same name)

Reply:
I set  those failing ones , still it did not help
` .put(new Info("system/Environment.h", "memory/MemoryType.h").objectify(false))` 
probably, there is something I'm doing wrong. I'll check it later